backend/buildeR.Common/Services/EmailService.cs

```diff
@@ -11,22 +11,33 @@ namespace buildeR.Common.Services
     
     public class EmailService : IEmailService
     {
-        public Task SendEmailAsync(List<string> emails, string subject, string message)
+        public Task SendEmailAsync(List<string> emails, string subject, string title, string body)
         {
 
-            return Execute(Environment.GetEnvironmentVariable("SENDGRID_API_KEY"), subject, message, emails);
+            return Execute(Environment.GetEnvironmentVariable("SENDGRID_API_KEY"), emails, subject, title, body);
+        }
+        private string CreateTemplate(string title, string body)
+        {
+            string htmlContent = @$"<table align=""center"" border=""1"" width=""600"" bordercolor=""#000000"" style=""color: #000000;""><tr><td align = ""center"" style = "" padding: 40px 0 30px 0;
```

Comment:
Please, make generation header and footer in separate methods? It should help with maintaining possible future changes.